src/App/DataSource/DataSource.php

```diff
@@ -26,4 +26,5 @@ public function getOptions();
     public function getInboundFields();
     public function getInboundFormId();
     public function getInboundFieldMappings();
+    public function isUserConfigurable();
```

Comment:
Added this to define which sources can be configured, and thus which appear in the API